build.gradle

```diff
@@ -50,7 +51,8 @@ ext {
   assertJCore = 'org.assertj:assertj-core:1.7.1'
   junit = 'junit:junit:4.12'
   mockitoCore = 'org.mockito:mockito-core:1.10.19'
-  robolectric = 'org.robolectric:robolectric:3.0'
+  robolectric = [core      : 'org.robolectric:robolectric:3.0',
+                 androidAll: 'org.robolectric:android-all:5.1.1_r9-robolectric-1']
```

Comment:
You're right. I wanted to make lib-android-robolectric a pure java project like core robolectric. `AttributeSetBuilder` requires Android `Context` so I added this dependency as provided there.